Add artemis (#388)
toitware/web-docs: docs/getstarted/device.mdx

import VSCode from "../_images/vscodehello.png";
import JaguarVideo from "../_images/146210503-24811800-bb26-4244-817d-6422b20e6786.png";

# Run on your device

This short guide shows how to install Toit, flash the Toit firmware on your [ESP32](https://www.espressif.com/en/products/socs/esp32),
setup your local development environment, and finally deploy a Toit application on your ESP32 over-the-air.

> It uses the open-source Jaguar service to enable live reloading when developing for the ESP32 and you

Reviewer: ```suggestion
It uses the open-source Jaguar service to enable live reloading when developing for the ESP32. You
```

> can watch a short video that shows how you can experience Jaguar on your ESP32 in two minutes:

Reviewer: ?
```suggestion
can watch this short two minute video to see the developer experience when using Jaguar on your ESP32:
```

> [<img src={JaguarVideo} alt="Video presentation of Jaguar" />](https://youtu.be/cU7zr6_YBbQ)

Reviewer: See comment for `docs/_images/14...`: might be nicer to use the youtube image instead. Unfortunately, it seems like the video is missing a maxresdefault option.

You might be able to generate it: https://stackoverflow.com/a/48178125

```

and then run it using:

``` sh
jag run hello.toit
```

At this stage you won't see anything unless you're still monitoring the serial output
(normally connected to your computer via USB). Use `jag monitor --attach` to start
listening to the serial output. The `--attach` flag is to make sure we don't reboot
the device. You should start seeing the "Hello, World!" messages printed out.

Jaguar is fast enough that it is possible to ask Jaguar to keep watching your Toit code
on disk and to *live reload* it when it changes. Simply write:

``` sh
jag watch hello.toit
```

and edit `hello.toit` or any of the files it depends on in your favorite editor.

## 5. Install a container

With `jag run` a program is only run once, and disappears when the device is rebooted. Jaguar supports
installing named containers that are automatically run when the system boots. They can be used
to provide services and implement drivers for peripherals. The services and drivers can be used by
applications and as such they form an instrumental part of the extensibility of a Jaguar based system.

To install the `hello.toit` program as a container, use:

``` sh
jag container install hello hello.toit
```

> After this the program will be rerun every time the device reboots. You can also list the currently installed

Reviewer: ?
```suggestion
After this the program is also run every time the device reboots. You can also list the currently installed
```
